e2e/cypress/support/api/role.js

```diff
@@ -160,6 +159,10 @@ export const defaultRolesPermissions = {
         'manage_outgoing_webhooks',
         'import_team',
         'join_public_teams',
+        'sysconsole_read_experimental_features',
+        'sysconsole_read_experimental_feature_flags',
+        'sysconsole_read_experimental_bleve',
+        'purge_bleve_indexes',
```

Comment:
Not necessarily as part of this PR. But these should probably be alphabetized. Its currently hard to read.